global.scss

```diff
@@ -2,6 +2,8 @@
   --brand-color-main: #5ab783;
   --brand-color-lighter: #{lighten(#5ab783, 20%)};
   --gray-border: #f2f2f2;
+  --dark: #000;
+  --light: #fff;
```

Comment:
być może to czepialstwo, ale nie jestem pewien czy nazywanie zmiennej `dark` i `light` na biały i czarny to dobry pomysł. Dałbym po prostu `black` i `white`

Reply:
Albo coś bardziej opisowego

Reply:
Nie no, myślę że `black` i `white` bedzie git

Reply:
Czy mogę zmienić na `black` i `white`?

Reply:
Nikt ci nie zabrania ;)